lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -7655,6 +7678,78 @@ class DecomposeAtenLinearOp : public OpRewritePattern<AtenLinearOp> {
 };
 } // namespace
 
+namespace {
+// Decompose `aten.bilinear` op into `aten._trilinear` and `aten.add` ops.
+class DecomposeAtenBilinearOp : public OpRewritePattern<AtenBilinearOp> {
```

Comment:
Can you please add some code-snippet as comment here to show how the decomposition looks like?

Reply:
Adding code-snippet as comment might make the code too verbose. I add some FileCheck tests to show how the decomposition would look like.